cmd/release/config/config.go

```diff
@@ -41,6 +41,25 @@ type RancherRelease struct {
 	SkipStatusCheck      bool   `json:"skip_status_check"`
 }
 
+type UIRelease struct {
+	UIRepoOwner   string `json:"ui_repo_owner" validate:"required"`
+	UIRepoName    string `json:"ui_repo_name"`
+	PreviousTag   string `json:"previous_tag"`
+	ReleaseBranch string `json:"release_branch" validate:"required"`
+	DryRun        bool   `json:"dry_run"`
+}
+
+type DashboardRelease struct {
+	DashboardRepoOwner   string `json:"dashboard_repo_owner" validate:"required"`
+	DashboardRepoName    string `json:"dashboard_repo_name"`
+	PreviousTag          string `json:"previous_tag"`
+	ReleaseBranch        string `json:"release_branch" validate:"required"`
+	Tag                  string
+	RancherUpstreamURL   string `json:"rancher_upstream_url" validate:"required"`
+	RancherReleaseBranch string `json:"rancher_release_branch" validate:"required"`
+	DryRun               bool   `json:"dry_run"`
```

Comment:
Same as before, but adding that I'll definitely make a PR updating the rancher cmd